src/lib/rest/clientSocketHttp.cpp

```diff
@@ -136,7 +136,8 @@ std::string sendHttpSocket
    const std::string&     content_type,
```

Comment:
I think the name "socket" is pretty obsolete in this and another files/functions. Maybe we should take the opportunity of this PR to change to something more meaninful.